src/content/reference/react/use-client.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: "'use client'"
+canary: true
 ---
 
-<Note>
+<Canary>
 
-These directives are needed only if you're [using React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks) or building a library compatible with them.
-
-</Note>
+`'use client'` available in the [Canary](https://react.dev/community/versioning-policy#canary-channel) release channel. It is needed only if you're [using React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks) or building a library compatible with them.
```

Comment:
still think its weird to talk about these directives being "available" in a certain channel since react doesn't interpret these. I would personally stick to only mentioning RSC in the text (but the callout icon/heading seems ok)

Reply:
I see, yeah makes sense for these ones.